docs/apache-airflow/authoring-and-scheduling/deferring.rst

```diff
@@ -213,6 +213,70 @@ This is particularly useful when deferring is the only thing the ``execute`` met
             # We have no more work to do here. Mark as complete.
             return
 
+Exiting deferred task from Triggers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ .. versionadded:: 2.10.0
+
+If you want to exit your task directly from the triggerer without going into the worker, you can specific operator level attribute ``end_from_trigger`` with the attributes to your deferrable operator other discussed above.
```

Comment:
Is the name `end_from_trigger` a convention or a requirement?

Reply:
It's a convention.